src/app/app.component.ts

```diff
@@ -125,13 +126,13 @@ export class AppComponent {
       return;
     }
 
-    const labelTitle = this.i18Service.transform('Angular Update Guide');
-    const labelBasic = this.i18Service.transform('Basic Apps');
-    const labelMedium = this.i18Service.transform('Medium Apps');
-    const labelAdvanced = this.i18Service.transform('Advanced Apps');
+    const labelTitle = this.i18Service.transform('Guide to update Angular versions');
```

Comment:
It's a guide to update Angular applications. Maybe:

```suggestion
Guide to update your Angular application
```